client/analytics/report/revenue.js

```diff
@@ -54,24 +54,70 @@ class RevenueReport extends Component {
 
 	/**
 	 * This function returns an event handler for the given `param`
+	 * @todo Move handling of this to a library?
 	 * @param {string} param The parameter in the querystring which should be updated (ex `page`, `per_page`)
 	 * @return {function} A callback which will update `param` to the passed value when called.
 	 */
 	onQueryChange( param ) {
-		return value => updateQueryString( { [ param ]: value } );
+		switch ( param ) {
+			case 'sort':
+				return ( key, dir ) => updateQueryString( { order_by: key, order: dir } );
```

Comment:
I'm wondering if `sort` should be the exposed term in the url. I'm looking at the designs to see if `order` will be used in the reports and I don't immediately see its use. However, unless you feel strongly about `order` having the meaning of table asc/desc, we could avoid a future collision of url query namespace.

Reply:
`sort` instead of `order`, you mean? I think we want to stay consistent with [core WP query naming](https://codex.wordpress.org/Class_Reference/WP_Query#Order_.26_Orderby_Parameters). We can use `order_id` if we need to, or more likely, `p`/`post`/`post_id`.

It also looks like the v3 API endpoints already support `order`/`orderby`, though I should update this from `order_by` to `orderby` for consistency.

Reply:
ah ok, yeah that makes sense